### PR TITLE
Removed the "Rebuild uses latest changes" messages.

### PR DESCRIPTION
**Pull request by BarryUnity**
Testing revealed that Rebuild _always_ uses the revision that the original build used. In that regard, the message saying that a rebuild will use the latest changes when rebuilding from certain builds was wrong. I don't think it needs a replacement.

### CODE REVIEW

File: www/templates/forms.html [hunk 1]
```diff
@@ -269,26 +269,10 @@
   </form>
 {% endmacro %}
 
-{% macro rebuild_build(rebuild_url, authz, exactly, ss, builderName, url) %}
+{% macro rebuild_build(rebuild_url, authz, ss, builderName, url) %}
```
Comment (BarryUnity) [inline www/templates/forms.html:272]: Can someone confirm that there's no consequences to removing this 'exactly' parameter? I couldn't find rebuild_build being used anywhere else in the code myself.

**Comment by campos-ddc** [comment]
That tooltip is misleading because the problems it exposes are actually related to the "Build latest" button, and not the "Rebuild" button.

I think that the confusion we have to solve here is the difference between those two. In slack I suggested `"Rebuid" will trigger a new build using the same revision as the last build. If you want to rebuild using the latest revision on branch X, press "Build Latest"`, what do you guys think about that?

**Review by mariangemarcano — CHANGES_REQUESTED** [review]
@BarryUnity it looks to me that exactly is not longer used, but I think that logic was added by @opdude so better double check it with him.

I dont see any .js changes so remove the changes pushed in .js files as they are not required for .html changes

**Comment by mariangemarcano** [comment]
@campos-ddc that message looks fine to me, I guess instead of "using the same revision as the last build" it should be "using the same revision as this build"

**Comment by opdude** [comment]
@mariangemarcano according to blame that came in from our merge from upstream. So no idea, looking at the code I think it's fine

**Comment by BarryUnity** [comment]
@campos-ddc is that not going overboard? I mean the message "To force a rebuild with the same source revisions and parameters used in this build, click the 'Rebuild' button" is still there. Is the button labelled 'build latest' really so vague that it needs an explanation for what it does?

**Review by mariangemarcano — APPROVED** [review]

**Pull request merged**